pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -190,6 +190,9 @@
     public static final double DEFAULT_BROKER_QUERY_LOG_MAX_RATE_PER_SECOND = 10_000d;
     public static final String CONFIG_OF_BROKER_TIMEOUT_MS = "pinot.broker.timeoutMs";
     public static final long DEFAULT_BROKER_TIMEOUT_MS = 10_000L;
+    public static final String CONFIG_OF_BROKER_SLOW_QUERY_LATENCY_THRESHOLD_MS =
+        "pinot.broker.slow.query.latency.thresholdMs";
+    public static final long DEFAULT_BROKER_SLOW_QUERY_LATENCY_THRESHOLD_MS = 500L;
```

Comment:
I would like to be conservative for the default number (may be 1second). Otherwise, this may suddenly start to add a lot of logs for some use cases whose queries are usually take > 500ms.

Reply:
Updated to 1s.